File: GRP28/totem/Atendimento_API/Models/Senha.cs
namespace Atendimento_API.Models
{
    public class Senha
    {
        public int id { get; set; }
        public string tipoSenha { get; set; }
        public DateTime? dataHoraEmissao { get; set; }
        public DateTime? dataHoraAtendimento { get; set; }
        public string statusAtendimento { get; set; }
        public int? tempoMinuto { get; set; }
        public string numeroSenha { get; set; }
    }
}

File: GRP28/totem/app_atendimento/src/app/shared/model/senha.ts
export interface Senha {
  id?: number;
  tipoSenha: string;
  dataHoraEmissao: Date | null;
  dataHoraAtendimento: Date | null;
  statusAtendimento: string;
  tempoMinuto?: number;
  numeroSenha: string;
}
